File: specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/examples-java/BlobServicesGet.java
/** Samples for BlobServices GetServiceProperties. */
public final class Main {
    /*
     * x-ms-original-file: specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/examples/BlobServicesGet.json
     */
    /**
     * Sample code: GetBlobServices.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void getBlobServices(com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .storageAccounts()
            .manager()
            .serviceClient()
            .getBlobServices()
            .getServicePropertiesWithResponse("res4410", "sto8607", com.azure.core.util.Context.NONE);
    }
}

File: specification/storage/resource-manager/Microsoft.Storage/stable/2022-05-01/examples-js/BlobServicesGet.js
const { StorageManagementClient } = require("@azure/arm-storage");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Gets the properties of a storage account’s Blob service, including properties for Storage Analytics and CORS (Cross-Origin Resource Sharing) rules.
 *
 * @summary Gets the properties of a storage account’s Blob service, including properties for Storage Analytics and CORS (Cross-Origin Resource Sharing) rules.
 * x-ms-original-file: specification/storage/resource-manager/Microsoft.Storage/stable/2022-05-01/examples/BlobServicesGet.json
 */
async function getBlobServices() {
  const subscriptionId = "{subscription-id}";
  const resourceGroupName = "res4410";
  const accountName = "sto8607";
  const credential = new DefaultAzureCredential();
  const client = new StorageManagementClient(credential, subscriptionId);
  const result = await client.blobServices.getServiceProperties(resourceGroupName, accountName);
  console.log(result);
}

getBlobServices().catch(console.error);
